### DISCUSSION TITLE
Where is the ErrorWithInfo interface?

### DISCUSSION THREAD

**Discussion by gmr458** [go-fuego/fuego#211] [General]
![20241024_12h07m01s_grim](https://github.com/user-attachments/assets/f9a5f172-80b5-4da4-b6a6-4d497718f83a)

**Comment by dylanhitt** [comment]
Hey @gmr458. That was removed with: https://github.com/go-fuego/fuego/pull/86/files

We now only have `ErrorWithStatus`. I'll update the doc.

Thank you
